plonetheme/barceloneta/theme/less/barceloneta-compiled.css

```diff
@@ -3922,6 +3942,25 @@ ul.picker__list {
 #content {
   margin-bottom: 48px;
 }
+#content-core a {
+  border-bottom: 1px solid #cccccc;
+}
+#content-core a:hover,
+#content-core a:focus {
+  border-bottom: none;
+}
+#content-core a:focus {
+  outline: thin dotted;
+  outline: 5px auto -webkit-focus-ring-color;
+  outline-offset: -2px;
+}
+#content-core .pat-structure a,
+#content-core .pat-resourceregistry a {
+  border-bottom: none;
```

Comment:
I'm sorry, I meant border-bottom. On those buttons. I assume it's related to these changes.